[Docs] Update security and spaces docs/screenshots (#105652)
elastic/kibana: docs/user/security/authentication/index.asciidoc

[[saml]]
==== SAML single sign-on

SAML authentication is part of single sign-on (SSO), a https://www.elastic.co/subscriptions[subscription feature]. This allows users to log
in to {kib} with an external Identity Provider, such as Okta or Auth0. Make sure that SAML is enabled and configured in {es} before setting

> it up in {kib}. See {ref}/saml-guide.html[Configuring SAML single sign-on on the Elastic Stack].

Author: General note for this and other SSO sections (OIDC, Kerberos): I changed them each to state "single sign-on (SSO)" at the beginning and then use SSO for the rest of the section.

[[oidc]]
==== OpenID Connect single sign-on

OpenID Connect (OIDC) authentication is part of single sign-on (SSO), a https://www.elastic.co/subscriptions[subscription feature]. Similar
to SAML, authentication with OIDC allows users to log in to {kib} using an OIDC Provider such as Google, or Okta. OIDC should also
be configured in {es}. For more details, see {ref}/oidc-guide.html[Configuring single sign-on to the {stack} using OpenID Connect].

> Enable OIDC authentication by specifying which OIDC realm in {es} should be used:

Reviewer: ```suggestion
Enable OIDC authentication by specifying which OIDC realm in {es} to use:
```

    realm: oidc1
--------------------------------------------------------------------------------

> If you want to use Third Party initiated SSO, configure your OpenID Provider to use `/api/security/oidc/initiate_login` as `Initiate Login URI`.

Reviewer: ```suggestion
To use third party initiated SSO, configure your OpenID Provider to use `/api/security/oidc/initiate_login` as `Initiate Login URI`.
```
